homework.md

```diff
@@ -4,107 +4,108 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12,240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HyperText Markup Language
+
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Adds style to a webpage. It's responsible for text style, positioning, color, size and much more.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+h1 {
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A computer programming language for the web. It can update and change both HTML and CSS and also has the ability to calculate,manipulate and validate data.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1) Helps you to keep track of your files and the changes you make to them, rather than you doing it manually yourself.
+2) Explore others' projects and get inspired or contribute to their project. It's a way for many people to collaborate on a project at  the same time.
+3) Markdown is a easy to use syntax for styling different forms of writing on Github.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+A fork is a copy of a repository. It allows you to make changes to a project without affecting the original version.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+It means you duplicate it and download everything that is in that repository.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+A pull request lets you tell others about the changes you pushed into a branch in a repository.
```

Comment:
👍  not only that, it is a request to make changes to the repository (by "pulling" the new code into the main branch)